GHzDACs/ghz_fpga_server.py

```diff
@@ -177,7 +177,7 @@
 ### BEGIN NODE INFO
 [info]
 name = GHz FPGAs
-version = 4.0.3
+version = 5.0.3
```

Comment:
OK. I had done 5.0.3 to indicate that it had been rebased off of 4.0.3, but that was more for myself than anything else.